app/javascript/components/Pages/AdoptionEvening/index.jsx

```diff
@@ -1,7 +1,6 @@
 import React from "react"
 import Markdown from 'markdown-to-jsx'
 import Layout from "../../Layout"
-import heroimage from "./bookevening.jpg"
```

Comment:
Are we using this image file elsewhere? If not I think we can probably delete it from the codebase, same with the other hero image.

We maybe want to think about how things look when you first set up the codebase and have no content in Fae as well, maybe we need some seed content or something like that. Not something to worry about with this PR though :)